src/core/client/framework/lib/validation.tsx

```diff
@@ -249,7 +261,23 @@ export const validatePercentage = (min: number, max: number) =>
 export const validateDeleteConfirmation = (phrase: string) =>
   createValidator((v) => v === phrase, DELETE_CONFIRMATION_INVALID());
 
-export const validateStrictURLList = createValidator((v) => {
+/**
+ * validateURLListStart is a validator that checks if a list of URLs all begin with 'http'.
+ */
+export const validateURLListStart = createValidator((v) => {
+  for (const url of v) {
+    if (!startsWith(url, "http")) {
+      return false;
+    }
+  }
+
+  return true;
+}, INVALID_URL_START());
+
+/*
+ * validateWeakURLList is validator that Checks if a list of URLs are all valid URLs.
+ */
+export const validateWeakURLList = createValidator((v) => {
```

Comment:
This should still keep the name `validateStrictURLList`, the strictness comes from the `URL_REGEX` that requires the protocol to be specified.